### PR TITLE
Refactor CardColumns to Accept and array of card objects as props

### PR DESCRIPTION
**Pull request by cherylli**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |

#### Have you updated the CHANGELOG.md file? If not, please do it.
yes

#### What is this change?
Refactor CardColumns to Accept and array of card objects as props

#### Were there any complications while making this change?
no

#### How did you verify this change?
run it locally

#### When should this be merged?
after review

### CODE REVIEW

File: components/ContactUsCards.js [hunk 1]
```diff
@@ -2,11 +2,39 @@ import contactCardStyles from '@/styles/ContactUsCards.module.scss';
...
+    linkText: 'Our Repository',
+  },
+];
+
 export default function ContactUsCards() {
   return (
     <div className={contactCardStyles.contactCards}>
       <RevealContentContainer>
         <CardsColumns
+          cards={cards}
```
Comment (mariana-caldas) [inline components/ContactUsCards.js:37]: Now we have the cards object here, can we remove lines 38-55, @cherylli ?

**Review by mariana-caldas — COMMENTED** [review]
I love the idea of using an object prop `card` to store the card's data instead of delegating that responsibility to the `CardsColumns`, @cherylli , great catch!

Other than that comment above, I think this is good to go.

File: components/ContactUsCards.js (hunk 1 above)
Comment (cherylli) [reply components/ContactUsCards.js:37]: yes, I forgot to remove them. thank you 🙂

**Review by mariana-caldas — APPROVED** [review]
Great refactor, @cherylli !

**Pull request merged**